pkg/keystore/keystore.go

```diff
@@ -123,7 +123,10 @@ func (ks *Keystore) MaskTrustedKeySystemPrefix(prefix, fingerprint string) (stri
 		return "", err
 	}
 	dst := path.Join(ks.LocalPrefixPath, acidentifier.String(), fingerprint)
-	return dst, ioutil.WriteFile(dst, []byte(""), 0644)
+	if err := ioutil.WriteFile(dst, []byte(""), 0644); err != nil {
+		return dst, err
```

Comment:
I think it is more idiomatic to `return "", err` here, and leave `dst` only for the final positive return.